File: launchdarkly/ruby/src/GetAllWebhooksExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::WebhooksApi.new.get_all_webhooks

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling WebhooksApi#get_all_webhooks: #{e}"
end

File: launchdarkly/python/src/GetAllWebhooksExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.WebhooksApi(api_client).get_all_webhooks()

        pprint(response)
    except ApiException as e:
        print("Exception when calling WebhooksApi#get_all_webhooks: %s\n" % e)
